src/app/pages/app-bar/page-wide-search/page-wide-search.component.html

```diff
@@ -0,0 +1,70 @@
+<mat-toolbar data-cy="pxb-toolbar" color="primary" class="main-header mat-elevation-z4 t" [class.hidden]="toggleSearch">
+    <mat-toolbar-row>
+        <button data-cy="toolbar-menu" mat-icon-button *ngIf="isSmall" (click)="toggleMenu()" style="margin-left: -8px">
+            <mat-icon>menu</mat-icon>
+        </button>
+        <div class="mat-title" [style.marginLeft.px]="isSmall ? 24 : 0">Page Search</div>
+        <pxb-spacer></pxb-spacer>
+
+        <div pxb-actions class="appbar-actions">
+            <button mat-icon-button>
+                <mat-icon style="margin: 0 8px">help</mat-icon>
+            </button>
+            <button mat-icon-button>
+                <mat-icon style="margin: 0 8px" [matBadge]="88" matBadgeColor="warn">notifications</mat-icon>
+            </button>
+            <button mat-icon-button>
+                <mat-icon>more_vert</mat-icon>
+            </button>
+        </div>
+    </mat-toolbar-row>
+</mat-toolbar>
+
+<mat-toolbar class="search-header" [class.active]="toggleSearch" *ngIf="isSmall">
+    <mat-toolbar-row style="padding: 0 16px">
+        <mat-icon>search</mat-icon>
+        <input
+            class="search-control"
+            type="text"
+            placeholder="Search Assets..."
+            [(ngModel)]="searchText"
+            #searchBar
+            data-cy="searchfield"
+        />
+        <button
+            *ngIf="searchText"
+            mat-icon-button
+            (click)="searchText = ''"
+            data-cy="search-close-btn"
+            style="margin-right: -8px"
+        >
+            <mat-icon>close</mat-icon>
+        </button>
+    </mat-toolbar-row>
+</mat-toolbar>
+<mat-divider *ngIf="isSmall"></mat-divider>
+
+<div class="content-body-container" [class.small]="isSmall">
+    <div class="content-body" [class.small]="isSmall">
+        <div *ngIf="!isSmall" class="desktop-search">
+            <pxb-spacer></pxb-spacer>
+            <mat-form-field appearance="standard" style="width: 160px">
+                <mat-icon matPrefix class="desktop-search-icon">search</mat-icon>
+                <input matInput type="text" placeholder="Search Asset" [(ngModel)]="searchText" />
```

Comment:
Make your desktop and mobile text match each other

```suggestion
                <input matInput type="text" placeholder="Search Assets..." [(ngModel)]="searchText" />
```

Reply:
Fixed 👍